document-viewer/src/main/java/org/emdev/common/content/ContentScheme.java

```diff
@@ -45,6 +45,9 @@ public String getResourceName(final ContentResolver cr, final Uri uri) {
                     final String attachmentFileName = c.getString(fileNameColumnId);
                     return LengthUtils.safeString(attachmentFileName, key);
```

Comment:
There should also be a `c.close()` added before this `return` statement, I think